### ISSUE TITLE
Add support for Doom Emacs

### ISSUE THREAD

**Issue by haditim** [donkirkby/live-py-plugin#278]
Hello and thanks for the awesome plugin.
I am here to report that the live-py-mode does not work correctly with Doom Emacs. The cursor decides to jump to the live window even if I am in the insert mode (evil).
# What I did
Any Python code would reproduce this. As an example:

```
import os

print(os.path.join('foo', 'bar')
```

# What happened
The cursor jumps to the *live-py-trace* buffer while typing if you delay the typing for more than one second.

# What I wanted to happen
Normal behavior as for Spacemacs or Vanilla Emacs.

# My environment
Describe the versions of everything you were using:
* Editor Doom Emacs
* Operating system Linux Manjaro
* Python 3.7

# Other feedback
I love this mode and being able to utilize it using Doom Emacs would be highly appreciated.

**Comment by donkirkby** [comment]
Thanks for the bug report, I haven't used [Doom Emacs] before. My first guess is that there's some plugin adding features to the cursor behaviour that's messing up `live-py-mode`. Any hints you can give me about Doom Emacs, how to list the active plugins and toggle them on and off would be appreciated.

[Doom Emacs]: https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org

**Comment by haditim** [comment]
Thanks for the reply. I think the packages can be retrieved using `doom/describe-package (SPC h p)`. But I am not aware of any interactive way to toggle packages off. The only way can be `(package! irony :disable t)` in the init file (SPC f p) and then run `~/.emacs.d/bin/doom sync` as it is mentioned in https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org#disabling-packages

**Comment by donkirkby** [comment]
OK, thanks for the hint. I'll try to reproduce the problem and then experiment with the package system.

**Comment by donkirkby** [comment]
OK, I followed the instructions linked above to install Doom Emacs, then installed `live-py-mode` by adding this to my `~/.doom.d/packages.el` file:

    (package! live-py-mode)

Finally, I ran `.emacs.d/bin/doom sync` to actually install it, and then edited a Python file.

    emacs scratch.py

I typed `C-z` to get out of evil mode, because I don't know how to launch `live-py-mode` in evil mode, then I typed `M-x live-py-mode`. I typed `M-x evil-normal-state` to go back to evil mode, and then I edited my Python code.

The cursor never left the main editor window, so I can't reproduce your problem. Please provide me with more information or try reproducing the problem on another system to see if we can learn more.

I tried it with Emacs 26.3 on Ubuntu 18.04 with Python 3.7.7.

**Comment by haditim** [comment]
Thanks a lot for taking time on this. I really appreciate it. 
Here is what I do (sorry for the flicker):
![gifcast_200629101207](https://user-images.githubusercontent.com/24245920/85990153-ddf06e80-b9f1-11ea-82bc-f225dc4f2385.gif)
I am using emacs version 26.3 on Linux 5.4.44-1-MANJARO with python 3.8.3

**Comment by donkirkby** [comment]
Thanks for recording the screencast. It's very unlikely that the Python version would cause this kind of problem. The Python process just writes the display to stdout. All the window handling takes place in Emacs.

Are you sure about that version number? I don't see anything like it in the [Manjaro downloads]. Also, how did you install Doom Emacs? If you're willing to write up a [Vagrant script] to reproduce the installation, that would be very very helpful, and I'd be much more likely to find a solution.

[Manjaro downloads]: https://manjaro.org/download/
[Vagrant script]: https://www.vagrantup.com/

**Comment by haditim** [comment]
Thanks a lot for the answer. 
I updated my Manjaro kernel, that's why it cannot be found on the ISO downloads. Today I freshly installed Manjaro on a virtual machine and after installing Doom Emacs and `live-py-mode` everything worked as they should. After cloning my config (from https://github.com/haditim/.doom.d) the problem showed up again. So now I am sure that there is a conflict between this and one of my settings/packages. I need to investigate more. 
Thanks again for taking time on this and sorry that I did not recognize this earlier.
